content/_data/changelogs/lts.yml

```diff
@@ -3604,6 +3604,311 @@
+      Add support for large buttons, hyperlinks styled as buttons and icon-only buttons.
+  - type: rfe
+    category: rfe
+    pull: 4663
+    authors:
+    - timja
+    message: |-
+      Restyle the help icon.
+  - type: rfe
+    category: rfe
+    pull: 4610
+    issue: 61478
+    authors:
+    - romenrg
+    message: |-
+      Improve styling of alert banners to be more visually appealing and to better match existing user interface components.
+      Alerts now fully cover the navigation bar while they are displayed instead of covering only a portion of the navigation bar.
+  - type: rfe
+    category: rfe
+    pull: 4555
+    issue: 61224
+    authors:
+    - romenrg
+    message: |-
+      Remove the unnecessary "monitor[s]" text next to the bell for a cleaner UI.
+      Change the colors of the notifications next to the bell to make them more noticeable.
+  - type: rfe
+    category: rfe
+    pull: 4633
+    authors:
+    - daniel-beck
+    message: |-
+      Improve the view shown when there are no jobs.
+
+  # Plugin manager improvements
+  - type: rfe
+    category: rfe
+    pull: 4513
+    issue: 61166
+    authors:
+    - daniel-beck
+    message: |-
+      Indicate when security issues would be addressed by an update in plugin manager.
+  - type: rfe
+    category: rfe
+    pull: 4580
+    authors:
+    - daniel-beck
+    - timja
+    - oleg-nenashev
+    message: |-
+      Don't show all available plugins by default; use search field to find plugins.
+  - type: rfe
+    category: rfe
+    pull: 4588
+    authors:
+    - daniel-beck
+    message: |-
+      Sort plugins by popularity on the <em>Available</em> plugin manager tab if the update site provides popularity data.
+  - type: rfe
+    category: rfe
+    pull: 4534
+    authors:
+    - daniel-beck
+    - fqueiruga
+    message: |-
+      Show plugin categories as labels in the plugin manager instead of grouping them into different table sections.
+  - type: rfe
+    category: rfe
+    pull: 4580
+    authors:
+    - daniel-beck
+    - timja
+    - oleg-nenashev
+    message: |-
+      Allow use of multiple space-separated filter terms in plugin manager.
+  - type: rfe
+    category: rfe
+    pull: 4591
+    authors:
+    - daniel-beck
+    - fqueiruga
+    message: |-
+      Allow linking to plugin manager URLs with pre-filled filter field.
+      Link labels in the plugin manager to pre-filtered lists.
+  - type: rfe
+    category: rfe
+    pull: 4535
+    authors:
+    - daniel-beck
+    message: |-
+      Show plugin release date in plugin manager.
+  - type: rfe
+    category: rfe
+    pull: 4589
+    authors:
+    - daniel-beck
+    message: |-
+      List plugins that failed to load on the Installed tab of the plugin manager.
+  - type: rfe
+    category: rfe
+    pull: 4584
+    authors:
+    - daniel-beck
+    message: |-
+      Highlight in the plugin manager when plugins are looking for new maintainers ("Adopt this plugin").
+
+  # System/Read improvements
```

Comment:
It would be good to link to the blog post for this somewhere https://www.jenkins.io/blog/2020/05/25/read-only-jenkins-announcement/

Reply:
That's an excellent idea.  I'll add it as a reference in the first item of that block, "Users with extended read permission get a more read-only looking UI".

Reply:
Done in 1b899b096bddebef93a9e76a9d831d368a7bd135.

Now looks like this:

![image](https://user-images.githubusercontent.com/156685/84571893-99a77200-ad53-11ea-8013-92a8680eed16.png)